src/go/rpk/pkg/out/in.go

```diff
@@ -137,3 +137,31 @@ func parseSpaceLines[T any](r io.Reader) ([]T, error) {
 	}
 	return vs, nil
 }
+
+// ParseTopicPartitions parses a topic:pa,rt,it,io,ns flag.
+func ParseTopicPartitions(list []string) (map[string][]int32, error) {
+	tps := make(map[string][]int32)
+	for _, item := range list {
+		split := strings.SplitN(item, ":", 2)
+		if len(split[0]) == 0 {
+			return nil, fmt.Errorf("item %q invalid empty topic", item)
+		}
```

Comment:
This conditional can be deleted, the conditional right below is doing what we want (and this conditional was meant to be that one).